docs/src/content/docs/guides/create-interactive-grant.mdx

```diff
@@ -138,6 +140,10 @@ Provided the end-user approves the grant, the authorization server:
 
 Add the authorization server’s interaction reference, as the `interact_ref` value, to the body of your continuation request.
 
+:::note[Continue request timing]
```

Comment:
Style choice: Instead of having this in a note, make it the first part of the overview for step 7.